packages/atomic/src/components/atomic-smart-snippet/atomic-smart-snippet-source.tsx

```diff
@@ -46,7 +47,7 @@ export class AtomicSmartSnippetSource implements InitializableComponent {
 
   render() {
     return (
-      <section aria-label={this.bindings.i18n.t('smart-snippet-source')}>
+      <Host>
```

Comment:
Do you need the `<Host>` if it has only one child?

Reply:
You don't, but it's not the only child.

Reply:
where is the second one??

Reply:
hidden by this button in GitHub:
![image](https://user-images.githubusercontent.com/54454747/168827111-e5f443cf-ab1f-4668-b240-fdafb35e9b66.png)

Reply:
aaah haha I promised I thought I had opened all of these buttons, but missed this one!! sorry

Reply:
I missed it too for a moment and looked like 5 times, wondering why there was only 1 element. Then I looked at the source code in Visual Studio Code and found it.